__tests__/app.test.js

```diff
@@ -275,4 +267,52 @@ describe('/api/articles/:article_id', () => {
       .expect(400)
       .then(({ body: { msg } }) => expect(msg).toBe('Invalid article ID'));
   });
+  describe('PATCH: /api/articles/:article_id', () => {
+    it('PATCH: responds with status 200 and the updated article when adding to votes', () => {
+      return request(app)
+        .patch('/api/articles/1')
+        .send({ inc_votes: 5 })
+        .expect(200)
+        .then(({ body: { article } }) => expect(article.votes).toBe(105));
+    });
+
+    it('PATCH: responds with status 200 and the updated article when removing votes', () => {
+      return request(app)
+        .patch('/api/articles/1')
+        .send({ inc_votes: -5 })
+        .expect(200)
+        .then(({ body: { article } }) => expect(article.votes).toBe(95));
+    });
+
+    it('PATCH: responds with status 400 and the updated article when removing votes', () => {
```

Comment:
Careful of your test description here. This would be status 200. You also have the same test above, so you can remove this one :)